### DISCUSSION TITLE
本仓库暂不接收Pull requests

### DISCUSSION THREAD

**Discussion by Wcowin** [Wcowin/Wcowin.github.io#3] [Announcements]
如果您对Mkdocs-Wcowin建设有好的idea，请移步[Wcowin / Mkdocs-Wcowin](https://github.com/Wcowin/Mkdocs-Wcowin/pulls),随时欢迎~

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JaneEyre0930** [comment]
我是简。
看到博主好厉害呀，
真想向你请教请教。
![201030231](https://user-images.githubusercontent.com/119165065/209401686-c21270cd-a5da-46eb-a036-f90a3b0bb0a0.jpg)

**Reply by Wcowin** [reply]
Hello, **Jane**. I sympathize with your study anxiety. I think there are many ways to overcome anxiety, such as reading books, listening to music and so on. I also welcome you to communicate with me again and hope you can learn more psychology knowledge on GitHub.

**Reply by JaneEyre0930** [reply]
你的主意听起来真不错。

**Comment by JaneEyre0930** [comment]
今天收到博主的邮件，不知怎么撤回了，是有什么东西要分享嘛？
